third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -35,6 +35,10 @@ use std::{
 pub struct LiveVarAnnotation(BTreeMap<CodeOffset, LiveVarInfoAtCodeOffset>);
 
 impl LiveVarAnnotation {
+    pub fn new(annot: BTreeMap<CodeOffset, LiveVarInfoAtCodeOffset>) -> Self {
+        Self(annot)
```

Comment:
Consider using this function in a couple of places currently where we directly create `LiveVarAnnotation` structs.